src/bootstrap/src/core/build_steps/test.rs

```diff
@@ -1445,7 +1445,7 @@ impl Step for RunMake {
 
     fn make_run(run: RunConfig<'_>) {
         let compiler = run.builder.compiler(run.builder.top_stage, run.build_triple());
-        run.builder.ensure(RunMakeSupport { compiler, target: run.target });
+        run.builder.ensure(RunMakeSupport { compiler, target: run.build_triple() });
         run.builder.ensure(RunMake { compiler, target: run.target });
```

Comment:
Does this need to be `run.build_triple()` instead too?

Reply:
No, this is the target for which the test will be run, rather than the target for which rmake.rs will be compiled.